values/cert-manager/cert-manager-raw.gotmpl

```diff
@@ -4,9 +4,12 @@
 {{- $p := $dns.provider }}
 {{- $certTpl := readFile "../../helmfile.d/snippets/certificate.gotmpl" }}
 {{- $vars := tpl (readFile "../../helmfile.d/snippets/domains.gotmpl") $v | fromYaml }}
+{{- $letsEncryptStage := printf "%s%s" "letsencrypt-" $cm.stage }}
+{{- $clusterIssuer := ternary "selfsigned-issuer" $letsEncryptStage (eq $v.cluster.provider "kind") }}
```

Comment:
Maybe we can simplify it and do not rely here on $v.cluster.provider "kind"

From values a user choose to use `cert-mnager.stage: selfsigned-issuer` and that should be enough to decide what kind of certificate the cert-manger is going to use